src/js/utils/JestUtil.js

```diff
@@ -88,6 +142,42 @@ const JestUtil = {
   },
 
   /**
+   * Generates a callback function to a reduce() call that will
+   * find and return the textContent of DOM element(s) matching
+   * the given selector
+   *
+   * @example <caption>How to use reduceTextContentOfSelector</caption>
+   * const TestUtils = require('react-addons-test-utils');
+   *
+   * var componentInstance = TestUtils.renderIntoDocument(
+   *     <MyTableComponent />
+   *   );
+   *
+   * var names = TestUtils.scryRenderedDOMComponentsWithClass(
+   *       componentInstance, 'table-column-name')
+   *       .reduce(reduceTextContentOfSelector(
+   *          '.collapsing-string-full-string'),
+   *          []
+   *       );
+   *
+   * @param {string} selector - The CSS selector to match children
+   * @returns {function} Returns a callback function to be passed on .reduce()
+   */
+  reduceTextContentOfSelector(selector) {
+    return function (strings, element) {
+      let stringContents = element.querySelectorAll(selector);
+      if (!stringContents) {
+        return strings;
+      }
+
+      Array.prototype.slice.call(stringContents).forEach(function (stringElement) {
```

Comment:
This is what I tried, but on our current node setup this method returns `undefined` if the selector did not match something.